### DISCUSSION TITLE
Previously banned users can't rejoin via invite link, even when unbanned via API and banned list in API does not list them...

### DISCUSSION THREAD

**Discussion by Sheldonfrith** [gram-js/gramjs#416] [Q&A]
I know this is not a gramjs issue, which is why I'm asking here (and I don't seem to have access to posting in the GramJS Chat telegram channel)...

Does anyone know how to allow a user to use invite links again after they've been banned once OTHER than manually removing them via the main telegram client?

**Comment by rojvv** [comment]
Isn’t the invite link expired whether due to usage limits or time limits?

**Reply by Sheldonfrith** [reply]
No this happens with any invite link, even freshly created ones. And when I manually remove them from the "Removed Users" list in the official client then they can use the same invite links again with no problem. Maybe what I should be asking is why are they showing up in the "removed/blocked users" list in the official client list when the api (via ChannelParticipantsBanned) shows them as not being blocked?
